File: fundamentos_objectivec/TPs/v2-assessment-questao-3/assessment-questao-3/Funcionario.m
//
//  Funcionario.m
//  assessment-questao-3
//
//  Created by Magno Valdetaro on 3/25/16.
//  Copyright © 2016 infnet. All rights reserved.
//

#import "Funcionario.h"

@implementation Funcionario

-(double)aumentoSalario:(double)isSalario {
    
    double resultado = isSalario * 1.07;
    return resultado;
}

@end

File: objectivec-swift/TPs/TP3/TP3/Funcionario.swift
//
//  Funcionario.swift
//  TP3
//
//  Created by Magno Valdetaro on 5/21/16.
//  Copyright © 2016 infnet. All rights reserved.
//

import Foundation

class Funcionario {
    func boasVindas<T>(nome : String, _ sobrenome : T) throws {
        
        if (nome == "") {
            throw ErrosMenu.EntradaInvalida
        } else {
            print("Seja bem vindo a nossa loja, \(nome) \(sobrenome)! Este são os itens a venda.")
        }
    }
}
